packages/backend/src/models/DashboardModel/DashboardModel.ts

```diff
@@ -597,6 +600,22 @@ export class DashboardModel {
         return this.getById(dashboardUuid);
     }
 
+    async updatePinning(
+        dashboardUuid: string,
+        dashboard: DashboardUnversionedFields,
+    ): Promise<Dashboard> {
+        const withSpaceId = dashboard.spaceUuid
+            ? { space_id: await getSpaceId(this.database, dashboard.spaceUuid) }
+            : {};
```

Comment:
We only need this part if we were moving the dashboard from one space to another.